src/components/forms/PlanForm/index.tsx

```diff
@@ -165,7 +165,7 @@ export const defaultInitialValues: PlanFormFields = {
   status: PlanStatus.DRAFT,
   taskGenerationStatus: 'False',
   teamAssignmentStatus: '',
-  title: '',
+  title: moment().format(DATE_FORMAT.toUpperCase()),
```

Comment:
Am not sure if this will solve the problem here. The populated plan title is usually a combination of location selected, FI classification and date. Not just date alone.